tests/test_scripts.py

```diff
@@ -31,6 +31,31 @@ def get_rule_path():
     return os.path.join(get_rules_path(), "lib", "allocate-memory.yml")
 
 
+@pytest.mark.parametrize(
+    # test detect_duplicate_features.py script for 3 cases
+    # - New rule isn't a valid rule
+    # - New rule file name doesn't ends with .yml
```

Comment:
I don't think we need to test these cases here. Let's focus on verifying the behavior like you do in case 3.